front/app/utils/yup/validateElementTitle.ts

```diff
@@ -0,0 +1,16 @@
+import { object } from 'yup';
+import validateMultilocForEveryLocale from './validateMultilocForEveryLocale';
+
+const validateElementTitle = (message: string) => {
+  return object().when('input_type', (input_type: string, schema) => {
+    if (input_type === 'page') {
+      return schema.test('input type is page', message, () => {
```

Comment:
Not sure exactly what is happening here so I could be wrong but if we are always returning `true`, then do we need to test?

Reply:
When I removed it, the function complained that the return type was incorrect, since we define "schema" here (I can take another look though):

`object().when('input_type', (input_type: string, schema)`

Reply:
🤔 Let us also add a unit test for the validator so we can capture some scenarios

Reply:
It seems we do need the schema test here? Since we're including the "schema" argument so we can assess the value of input_type, the return needs to be in a "SchemaLike" format or it complains.